template.adoc

```diff
@@ -49,21 +49,34 @@ You'll explore how to....
 // You'll next add the various sections that are needed for a particular guide.
 
 
-== Getting Started
 // Add this getting started section to your guide if it is applicable. Use the following include to pull in the git clone instructions from the guides-common repo.
-
+== Getting started
 include::{common-includes}/gitclone.adoc[]
 
+// This is a subsection of the "Getting started" section above. It should briefly walk the user through
+// how to setup everything in the "finish" directory and try out the finished version of what
+// they will be building.
+=== Try what you'll build
+
+
 
-== Building the application
-// Add this Building the application section to your guide if it is applicable.
+// Add this section to your guide if it is applicable.
 // Put this section in the appropriate order in your guide.
-// Use the following include to pull in the Maven build instructions from the guides-common repo.
+== Building and running the application
 
+// Use the following include to pull in the Maven build instructions from the guides-common repo.
 include::{common-includes}/mvnbuild.adoc[]

```

Comment:
Use the following "include"?  I think using include as a noun may cause some confusion. Use the following example? code? text? command?